app/models/guidance_group.rb

```diff
@@ -30,8 +33,19 @@ class GuidanceGroup < ActiveRecord::Base
   # has_and_belongs_to_many :guidances, join_table: "guidance_in_group"
 
 
+  # ===============
+  # = Validations =
+  # ===============
+
+  validates :name, presence: { message:  PRESENCE_MESSAGE }
+
+  validates :org, presence: { message: PRESENCE_MESSAGE }
+
+  validates :optional_subset, inclusion: { in: BOOLEAN_VALUES,
+                                           message: INCLUSION_MESSAGE }
 
-  validates :name, :org, presence: {message: _("can't be blank")}
+  validates :published, inclusion: { in: BOOLEAN_VALUES,
+                                     message: INCLUSION_MESSAGE }
 
```

Comment:
Do we also want to check uniqueness for name for a given org? `validates :name, uniqueness: { message: UNIQUENESS_MESSAGE, scope: :org }` to keep names unique across an org.

Reply:
Added